packages/block-editor/src/components/block-mover/mover-description.js

```diff
@@ -65,13 +66,13 @@ export function getBlockMoverDescription(
 
 	if ( dir > 0 && ! isLast ) {
 		// Moving down.
-		const movementDirection = getMovementDirection( 'down' );
+		const movementDirection = getMovementDirection( 'down', orientation );
 
 		if ( movementDirection === 'down' ) {
 			return sprintf(
 				// translators: 1: Type of block (i.e. Text, Image etc), 2: Position of selected block, 3: New position
 				__(
-					'Move %1$s block from position %2$d down to position %3$d'
+					`Move %1$s block from position %2$d down to position %3$d`
```

Comment:
Was the change here needed?

Reply:
Oh, that's my mistake 😢
I have fixed it.